awx/main/migrations/0164_remove_inventorysource_update_on_project_update.py

```diff
@@ -1,24 +1,13 @@
 # Generated by Django 3.2.13 on 2022-06-21 21:29
 
 from django.db import migrations
-import logging
-
-logger = logging.getLogger("awx")
```

Comment:
If someone had a large number of inventory sources are there any concerns that the logging might slow down the migration?

Reply:
Possibly, though my main concerns with removing it are,

1. our loggers make use of the database, except under circumstances that I don't entirely trust
2. migrations are usually run unattended, so it feels like this is just going to be spamming the logs

Reply:
And of course the obvious: if we are attempting to avoid the side-effects involved with executing `.save()` on each object, we don't really have an opportunity to individually log each object.

Reply:
I thought we were pretty confident that migrations use the "default" (meaning file-based) settings. This might not respect external logging settings, but it's something better than nothing.

you can still log the return value of bulk updates.

```python
In [2]: Project.objects.filter(name='foobar').update(scm_update_on_launch=True)
Out[2]: 0
```

I have done this in several migrations before. I value the feedback logging gives, but this is more of a style topic than anything else, and I wouldn't ever say it's a blocker.